CharlieBackend.Data/Repositories/Impl/Interfaces/IDashboardRepository.cs

```diff
@@ -0,0 +1,43 @@
+﻿using CharlieBackend.Core.DTO.Dashboard;
+using CharlieBackend.Core.DTO.StudentGroups;
+using CharlieBackend.Core.Entities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.Data.Repositories.Impl.Interfaces
+{
+    public interface IDashboardRepository
+    {
+        public Task<List<long>> GetGroupsIdsByCourceIdAsync(long courceId, DateTime startDate, DateTime finishDate);
+
+        public Task<List<long>> GetStudentsIdsByGroupIdAsync(long studentGroupId);
```

Comment:
This method is not needed - GetStudentsIdsByGroupIdsAsync can be used instead

Reply:
Sorry, its from previous versions of Dashboard